src/Stryker.Core/Stryker.Core/Mutants/Mutant.cs

```diff
@@ -30,6 +31,10 @@ public class Mutant : IMutant
 
     public string DisplayName => $"{Id}: {Mutation?.DisplayName}";
 
+    public string ReplacementText { get; init; }
+
+    public FileLinePositionSpan OriginalLocation { get; init; }
```

Comment:
This is one of the things i'm not especially happy about. This is needed because the html report would otherwise be wrong, because the mutation is applied to new source that is injected and the report does not show this source. I originally had derived classes of `GeneratedRegexMutant` and `GeneratedRegexMutation` to not have these properties on regular mutants, but that became too complex. All existing mutations do not set these values, and they default to what they would have been before in JsonMutant, and that is the only place these properties are used.